tools/topology/sof-icl-rt711-rt1308-rt715-hdmi.m4

```diff
@@ -185,9 +195,6 @@ DAI_CONFIG(ALH, 3, 1, SDW0-Capture)
 #ALH SDW1 Pin2 (ID: 2)
 DAI_CONFIG(ALH, 0x102, 2, SDW1-Playback)
 
-#ALH SDW2 Pin2 (ID: 3)
-ifdef(`MONO', `', `DAI_CONFIG(ALH, 0x202, 3, SDW2-Playback)')
```

Comment:
These two changes for SDW2 look like they are fixing another issues - separate patches ?

Reply:
> These two changes for SDW2 look like they are fixing another issues - separate patches ?

Yes, done